server.js

```diff
@@ -1956,6 +1957,7 @@ function objectWebServer() {
                         } else {
                             console.log('successfully wrote settings hardwareInterface: ' + interfaceName);
                             callback(true);
+                            hardwareAPI.pushSettingsToGui(interfaceName, existingSettings);
```

Comment:
For safety it would be a good idea to add a mock of pushSettingsToGui to the mobile hardwareAPI object https://github.com/ptcrealitylab/vuforia-spatial-edge-server/blob/master/libraries/mobile/hardwareInterfaces.js

```
pushSettingsToGui: function() {
}
```

Reply:
Oh, I forgot about that. Good catch